packages/node_modules/@ciscospark/widget-space-demo/src/components/demo-widget-space/index.js

```diff
@@ -229,8 +237,30 @@ class DemoWidgetSpace extends Component {
                 </RadioButtonGroup>
               </div>
 
+              <h3> Widget Initial Activity </h3>
+              <div className={classNames(styles.select)}>
+                <RadioButtonGroup
+                  aria-label="Initial Activity"
+                  name="initialActivity"
+                  onChange={this.handleInitialActivityChange}
+                  valueSelected={this.state.initialActivity}
+                >
+                  <RadioButton
+                    aria-label="Message"
+                    label="Message"
+                    value="message"
+                  />
+                  <RadioButton
+                    aria-label="Meet"
+                    label="Meet"
+                    value="meet"
+                  />
+                </RadioButtonGroup>
+              </div>
+
               <div>
                 <h3> Widget Activities </h3>
+                {displayActivityWarning && (<div> Warning: The initial activity must be selected </div>)}
```

Comment:
One thing that would make this a bit more like the rest of our JSX code is to put the html on a new line. Not sure if we have an eslint rule possible for this or not.